test2/faces40/pom.xml

```diff
@@ -32,8 +32,16 @@
     <name>Mojarra ${project.version} - Test - Faces 4.0</name>
 
     <modules>
-        <module>ajax</module>
+
+    	  <module>ajax</module>
+        <module>cdi</module>
+        <module>extensionless-mapping</module>
+        <module>javaPage</module>
+        <module>javaPageWithMetadata</module>
+        <module>selectItemGroups</module>
```

Comment:
```suggestion
```

Reply:
Thanks, there was a merge error there. I already fixed it locally as obviously the tests completely failed. But I can just as well accept those changes. Thanks again!